gulpfile.js

```diff
@@ -29,7 +30,17 @@ elixir(function(mix) {
     		'public/css/app.css'
     	], {
     		liveCSS: true
-    	});
+    	})
+        .karma({
+            jsDir:[
+                'public/js/vendor.js',
+                'node_modules/angular-mocks/angular-mocks.js',
+                'public/js/partials.js',
+                'public/js/app.js',
+                'tests/angular/**/*spec.js'
```

Comment:
I would propose another solution, because if the application is bigger I think having the test files in the same folders as your source code it would be difficult to maintain. I think a better idea wold be to create the same folder structure in the test folder for example: 

```
angular/
------services/
------------some.service.js
```
and the test would be :
```
tests/
------angular/
-----------services/
----------------some.service.spec.js
```

This could be done for all the existing js file: 'config' ,'filters' etc.
Tell me what you think about this approach.